kge/dataset.py

```diff
@@ -91,6 +93,20 @@ def load(config: Config, preload_data=True):
                 dataset.split(split)
         return dataset
 
+    @staticmethod
+    def load_meta(config: Config, meta_checkpoint: Dict, preload_data=True) -> Dataset:
+        """Loads a dataset and overrides meta data."""
+        dataset = Dataset.load(config, preload_data)
+        dataset._meta.update(meta_checkpoint)
+        return dataset
+
+    def save_meta(self, keys: List[str]):
```

Comment:
Use save_to_checkpoint as well, but accept a meta argumetn. If not given, nothing is saved.

Reply:
done, now called save_to